src/_layouts/default.html

```diff
@@ -5,9 +5,9 @@
 
   <body class="body body--{{ page.layout }}">
 
-    {%- include header.html -%}
+    {%- include header2.html -%}
```

Comment:
The include directive is run when Jekyll generate the HTML. Not when we're serving the website. So there's no way to A/B test this as-is.

One way to do this is to render BOTH headers and display:none one when a css class is set on the body. This is pretty easy to toggle with optimize

Reply:
Ohhhh, that makes sense now. Will do...

Reply:
Or, actually, I could use your help to make that small change once I merge this PR, just so I can continue focusing on the Landing Page layout.

Reply:
ok